tornado_json/schema.py

```diff
@@ -1,9 +1,12 @@
 import json
+
```

Comment:
I'd prefer to not have these additional newlines (this and the two below it). I don't think they add anything to the readability of the code.